src/main/java/net/neoforged/neoforge/client/model/StandardModelParameters.java

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.client.model;
+
+import com.google.gson.JsonDeserializationContext;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.mojang.math.Transformation;
+import java.util.HashMap;
+import java.util.Map;
+import net.minecraft.client.renderer.block.model.ItemTransforms;
+import net.minecraft.client.renderer.block.model.TextureSlots;
+import net.minecraft.client.renderer.texture.TextureAtlas;
+import net.minecraft.client.resources.model.UnbakedModel;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.util.GsonHelper;
+import net.neoforged.neoforge.client.RenderTypeGroup;
+import org.jetbrains.annotations.Nullable;
+
+/**
+ * Wrapper around all standard top-level model parameters added by vanilla and NeoForge except elements.
+ * <p>
+ * For use in custom model loaders which want to respect these properties but create the quads from
+ * something other than the vanilla elements spec.
+ */
+public record StandardModelParameters(
```

Comment:
Should this be called `StandardModelProperties` to match `NeoForgeModelProperties`?

Reply:
On one hand I agree, on the other hand I'm concerned that it would be confusing because one holds actual data and the other only contains some keys.